src/assets/htmx.md

```diff
@@ -0,0 +1,347 @@
+---
+title: htmx
+authorName: Niraj Acharya
+authorAvatar: https://avatars.githubusercontent.com/u/122071597?v=4
+authorLink: https://github.com/nirajacharya2
+createdAt: Sep 29, 2023
+tags: js, html, go, htmx
+banner: https://raw.githubusercontent.com/JankariTech/blog/master/src/imgs/fallback_banner.png
+---
+
+Htmx is ui library written in JavaScript that extends the features of html. It can be extended to make AJAX requests, do CSS transitions, connect to WebSockets and handle Server Sent Events inside a html document with just some delacritive html attributes.
```

Comment:
```suggestion
htmx is ui library written in JavaScript that extends the features of html. It can be extended to make AJAX requests, do CSS transitions, connect to WebSockets and handle Server Sent Events inside a html document with just some delacritive html attributes.
```

if htmx is generally written with a small h, you can also use small h at the beginning of a sentence because it's a name. putting names in _italic_ or using quote like this `htmx` helps the reader to better understand what are names or commands within a text